libs/model-ad/model-details/src/lib/model-details.component.html

```diff
@@ -1,10 +1,14 @@
 <div [class]="{ 'model-details': true, loading: isLoading }">
   @if (isLoading) {
     <div id="loading">
-      <explorers-loading-icon />
+      <explorers-loading-icon
+        [colorOutermost]="loadingIconColors.colorOutermost"
+        [colorCentral]="loadingIconColors.colorCentral"
+        [colorInnermost]="loadingIconColors.colorInnermost"
+      />
```

Comment:
Something we could consider is creating a model-ad-loading-icon component so we don't have to set the loadingIconColors everywhere.  It would effectively wrap this component with the needed props.